api/proto/config/acipherinfo.proto

```diff
@@ -1,4 +1,5 @@
 // Copyright(c) 2017-2018 Zededa, Inc.
+// really? 2017-2018?
```

Comment:
Good point -- any chance you can fix those too as part of this?

Reply:
I'd kind of like to avoid doing that in this patch in case this gets backed out for whatever reason.

Reply:
@aw-was-here Is there a copyright check tool in Yetus?
Checking that 1) there is a Copyright and 2) that the year has been updated wouldn't be a bad idea.

Reply:
It doesn't have that sort of test. Thinking about it, I suspect there are quite a few edge cases though. Imported code, when does a copyright get updated, how many different forms can a copyright notice take, etc.